oneview_redfish_toolkit/api/subscription_collection.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import collections
+from oneview_redfish_toolkit.api.redfish_json_validator \
+    import RedfishJsonValidator
+
+
+class SubscriptionCollection(RedfishJsonValidator):
+    """Creates a Subscription Collection Redfish dict
+
+        Populates self.redfish with some hardcoded SubscriptionCollection
+        values
+    """
+
+    SCHEMA_NAME = 'EventDestinationCollection'
+
+    def __init__(self):
+        """SubscriptionCollection constructor.
+
+            Populates self.redfish with hardcoded SubscriptionCollection
+            values.
+        """
+        super().__init__(self.SCHEMA_NAME)
+
+        self.redfish["@odata.type"] = \
+            "#EventDestinationCollection.EventDestinationCollection"
+        self.redfish["Name"] = "Event Subscriptions Collection"
+        self.redfish["Members@odata.count"] = 1
+        self.redfish["Members"] = list()
+        self.redfish["Members"].append(collections.OrderedDict())
+        # This information is currently hard-coded
+        self.redfish["Members"][0]["@odata.id"] = \
```

Comment:
Not hard coded, please!!!! :(
At this point we can define at lease empty dictionaries in utils to hold collections of subscriptions and iterate on them here... they'll be empty on demos until we implement add subscription endpoints but that's life...

Reply:
Ok, I'm going to change that.